package.json

```diff
@@ -1,19 +1,29 @@
 {
   "name": "private-box2",
   "description": "new private message format for ssb",
-  "version": "1.0.0",
+  "version": "0.1.0",
   "homepage": "https://github.com/ssbc/private-box2",
   "repository": {
     "type": "git",
     "url": "git://github.com/ssbc/private-box2.git"
   },
+  "scripts": {
+    "generate": "node test/generate.js",
+    "test": "npm run test:js && npm run lint",
+    "test:js": "tape 'test/**/*.test.js' | tap-spec",
+    "lint": "eslint ."
```

Comment:
@dominictarr I've added the most minimal `.eslintconfig` I can.
Basically I ran it and disabled (stupid) rules till everything is passing (e.g. no-camel-case, stupid ++ rules). The purpose of this is to catch unused-variables and dangerous things, not to fix indenting.

My policy here is to disable rules that are in the way (notice we don't bother linting the `test/` folder - `.eslintignore`).

This is an experiment in a compromise which I hope keeps the code easy to maintain (no unused variables accidentally hanging around), while also respecting the need to do layout things to improve readability sometimes (e.g. do indenting and camel_case sometimes, and multi-line spaces)